configure.ac

```diff
@@ -41,7 +41,7 @@ AS_CASE([$host_os],
 # [3]: https://lists.gnu.org/archive/html/automake/2004-01/msg00013.html
 MKDIR_P=${MKDIR_P:-install -d -m 0755}
 
-AM_INIT_AUTOMAKE([1.13 -Wall -Werror foreign subdir-objects])
+AM_INIT_AUTOMAKE([1.13 -Wall -Werror tar-ustar foreign subdir-objects])
```

Comment:
IMO it would be good if we could stay with the default tar format. If we install into say `lib/embed` instead of e.g. `lib/lib/python3.7/site-packages` will that work?

Reply:
I tried using a slightly shorter path name but it breaks even if you use `lib/modules` by a long shot on some files. I can double check if that would work with the no compile flag, as that may shorten some paths.